book/src/drivers.md

```diff
@@ -1,61 +1,61 @@
 # Drivers
 The following are a set of CSI driver which can be used with Kubernetes:
 
-> NOTE: If you would like your driver to be added to this table, please open a pull request in [this repo](https://github.com/kubernetes-csi/docs/pulls) updating this file.
+> NOTE: If you would like your driver to be added to this table, please open a pull request in [this repo](https://github.com/kubernetes-csi/docs/pulls) updating this file. Other Features allows to be filled in Raw Block, Snapshot, Volume Expansion and Volume Cloning. If driver did not implement any Other Features, please leave it blank.
```

Comment:
typo: allowed

Also can we shorten feature names by dropping the "Volume"? So just "Expansion" and "Cloning"

Reply:
> typo: allowed
> 
> Also can we shorten feature names by dropping the "Volume"? So just "Expansion" and "Cloning"

Done